packages/interaction-ring/src/InteractionRing.tsx

```diff
@@ -96,16 +96,6 @@ function computeStyles({
   };
 }
 
-const baseContentStyle = css`
-  width: 100%;
```

Comment:
The `TextArea` has CSS `width` and `min-height` defined there, so it should be fine with this change. Does this affect other components? (I don't see this applied to the `leafygreen-card`)

Reply:
It's used in a number of other components (not card, however). I'm gonna get a second set of eyes on this to verify this week.

Reply:
Doesn't look like this should break anything, and should be a simple fix in case it does